src/components/FeatureViews/Accounts/TabComponents/InstructorSchedule.js

```diff
@@ -26,6 +28,27 @@ const Schedule = ({instructorID}) => {
                     : "Loading...",
             })), [courses, sessions, instructorID]);
 
+    const OOO = useMemo(() => instructor
+        ? Object.values(instructor.schedule.time_off)
+            .map(({all_day, description, start, end}) => {
+                const dayAfterEnd = new Date(end);
+                // since the end date for allDay events is EXCLUSIVE
+                // must add one day to include the end specified by user
+                if (all_day) {
+                    dayAfterEnd.setDate(dayAfterEnd.getDate() + 1);
+                }
+                return {
+                    "allDay": all_day,
+                    "end": dayAfterEnd,
+                    start,
+                    "title": description || "Out of Office",
+                };
+            })
+        : []
+    , [instructor]);
+
+    const allDayOOO = useMemo(() => OOO.some(({allDay}) => allDay), [OOO]);
```

Comment:
these aren't displaying for me. Also, be sure to color the OOO event the same color as the profile colors using @jsflca2012 's function
<img width="1521" alt="Screen Shot 2020-03-01 at 4 02 54 PM" src="https://user-images.githubusercontent.com/12959959/75636735-229c0280-5bd6-11ea-807c-8ba07a37e9f1.png">

Reply:
you can look for `stringToColor()` at accountUtils.js